packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -55,6 +67,7 @@ class Layout extends React.Component {
             onRequestClose={this.toggleNav}
           />
           <main id="main-container" className={cx('main-container', this.state.isMobileNavOpen && 'nav-open')}>
+            <ResponsiveElement defaultElement={overlay} responsiveTo="window" medium={<div />} />
```

Comment:
do we need responsive element? You could probably just always render the overlay

Reply:
we do. if we always render the overlay and the navOpen flag is set to true (for example resizing from mobile to desktop while nav is open), it breaks desktop by allowing scroll and forces the user to click main content once to close the nav so they can interact with the content itself.
<img width="1282" alt="image" src="https://user-images.githubusercontent.com/30411845/30220366-21c26cea-948d-11e7-9439-b8cad1d1f01c.png">

Reply:
i tried setting css properties of visibility: hidden and display: none to the overlay & tried setting the same properties on a wrapper around the overlay but neither affected it.

Reply:
ok